kolibri/plugins/user/assets/src/vue/profile-page/index.vue

```diff
@@ -1,50 +1,105 @@
 <template>
 
   <div class="profile-page">
-    <div v-if="hasPrivilege('username')" class="input-field">
-      <label>Username</label>
-      <input autocomplete="username" id="username" type="text"/>
+    <div v-if="error" class="error ">
+      {{errorMessage}}
     </div>
-    <div v-if="hasPrivilege('name')" class="input-field">
-      <label>Name</label>
-      <input autocomplete="name" id="name" type="text"/>
-    </div>
-    <div v-if="hasPrivilege('password')"  class="input-field">
-      <label>Password</label>
-      <input autocomplete="new-password" id="password" type="password"/>
-    </div>
-    <div v-if="hasPrivilege('password')"  class="input-field">
-      <label>Confirm Password</label>
-      <input autocomplete="new-password" id="confirm-password" type="password"/>
-    </div>
-    <div v-if="hasPrivilege('delete')"  class="input-field">
-      <span class="advanced-option">Delete Account</span>
-    </div>
-    <div class="input-field">
-      <button>Update Profile</button>
+    <div v-if="success" class="success">
+      {{$tr('success')}}
     </div>
+    <form @focus.capture="" @submit.prevent="submitEdits">
+      <div v-if="hasPrivilege('username')" class="input-field">
```

Comment:
the `hasPrivilege` method is unnecessary. Just reference the values directly on the `privileges` getter:

```html
<div v-if="privileges.username" class="input-field">
```

We may want to rename these fields eventually.